src/coffee/io/reader.coffee

```diff
@@ -6,24 +6,18 @@ iconv = require 'iconv-lite'
 fs = Promise.promisifyAll require('fs')
 Excel = require 'exceljs'
 
-DEBUG = false
-debugLog = if DEBUG then console.log else _.noop
```

Comment:
Sry @hisabimbola  - was reacting on Siilwyn approve. I removed it because that `if` was dropping branches coverage. Also there was one spec file removed from tests plus I added test for unsupported encoding... Should I return those debug logs back?